server/protocols/webadmin/webadmin-mailbox-deleted-message-vault/src/test/java/org/apache/james/webadmin/vault/routes/DeletedMessagesVaultRoutesTest.java

```diff
@@ -76,12 +93,19 @@
 import io.restassured.RestAssured;
 import io.restassured.filter.log.LogDetail;
 import reactor.core.publisher.Flux;
+import reactor.core.publisher.Mono;
 
 class DeletedMessagesVaultRoutesTest {
 
+    private static final String MATCH_ALL_QUERY = "{" +
```

Comment:
I find that confusing. why don't we use an empty object instead? Using a combinator for an empty list looks like a hack

And BTW, we want to forbid this query anyway for limiting information disclosure.

Reply:
> And BTW, we want to forbid this query anyway for limiting information disclosure.

Will be discussed then addressed in a separate task

Reply:
> I find that confusing. why don't we use an empty object instead? Using a combinator for an empty list looks like a hack

On a theorical basis it works: no criteria means everything is valid...